src/main/java/com/hrznstudio/galacticraft/server/command/GalacticraftCommands.java

```diff
@@ -23,119 +23,251 @@
 package com.hrznstudio.galacticraft.server.command;
 
 import com.hrznstudio.galacticraft.api.celestialbodies.CelestialBodyType;
+import com.mojang.brigadier.Command;
 import com.mojang.brigadier.builder.LiteralArgumentBuilder;
 import com.mojang.brigadier.context.CommandContext;
 import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import com.mojang.brigadier.tree.LiteralCommandNode;
 import net.fabricmc.fabric.api.command.v1.CommandRegistrationCallback;
 import net.minecraft.block.Block;
-import net.minecraft.block.BlockState;
-import net.minecraft.block.Blocks;
+import net.minecraft.command.argument.BlockPosArgumentType;
 import net.minecraft.command.argument.DimensionArgumentType;
 import net.minecraft.command.argument.EntityArgumentType;
 import net.minecraft.entity.Entity;
 import net.minecraft.server.command.CommandManager;
 import net.minecraft.server.command.ServerCommandSource;
-import net.minecraft.server.command.TeleportCommand;
 import net.minecraft.server.network.ServerPlayerEntity;
 import net.minecraft.server.world.ServerWorld;
 import net.minecraft.text.LiteralText;
 import net.minecraft.text.Style;
 import net.minecraft.text.TranslatableText;
 import net.minecraft.util.Formatting;
 import net.minecraft.util.math.BlockPos;
-import net.minecraft.util.math.Vec3d;
-import net.minecraft.world.Heightmap;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.util.registry.RegistryKey;
+import net.minecraft.world.dimension.DimensionType;
 
 import java.util.Collection;
+import java.util.HashMap;
+import java.util.UUID;
 import java.util.function.Consumer;
 
 /**
  * @author <a href="https://github.com/StellarHorizons">StellarHorizons</a>
  */
 public class GalacticraftCommands {
 
+    private static final HashMap<UUID,Long> GCR_HOUSTON_TIMERS = new HashMap<>();
+    private static final long GCR_HOUSTON_TIMER_LENGTH = 12L * 1000L; // in milliseconds
+
     public static void register() {
         CommandRegistrationCallback.EVENT.register((commandDispatcher, b) -> {
-            commandDispatcher.register(LiteralArgumentBuilder.<ServerCommandSource>literal("dimensiontp")
+
+            commandDispatcher.register(
+                    LiteralArgumentBuilder.<ServerCommandSource>literal("gcrhouston")
```

Comment:
Use `CommandManager#literal` instead of `LiteralArgumentBuilder#literal` to avoid explicitly specifying generics